oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+autoAssign() {
+  curl -X POST "https://api.github.com/repos/microsoft/Oryx/issues/$3/assignees" -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $2" -d "{\"assignees\": ["\"$1\""]}"
```

Comment:
nit: let's assign the arguments to this function to variables so it's clear what each argument represents when provided to the `POST` request here